target/linux/brcm63xx/dts/dsl-2750u-c1.dts

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "bcm6328.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "D-Link DSL-2750u rev C1";
+	compatible = "d-link,dsl-2750u-c1", "brcm,bcm6328";
+
+	chosen {
+		bootargs = "rootfstype=squashfs,jffs2 noinitrd console=ttyS0,115200";
+		stdout-path = "serial0:115200n8";
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&pinctrl 23 1>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&pinctrl 24 1>;
+			linux,code = <KEY_WPS_BUTTON>;
+			debounce-interval = <60>;
+		};
+	};
+
+	gpio-leds {
```

Comment:
@naseef  Could you please change this to generic name `leds` as suggested by [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist)
>  5. The name of a node should reflect the function of the device and not its model. Examples for generic node names can be found in Section 2.2.2 Generic Names Recommendation

While at it, could you please add newline between LED nodes:

```
                inet_red {
			label = "DSL2750U-C1:red:inet";
			gpios = <&pinctrl 1 0>;
		};

		power_red {
			label = "DSL2750U-C1:red:power";
			gpios = <&pinctrl 4 0>;
		};
```

```

Reply:
And probably switch to lowercase board name in LED label `dsl-2750u-c1:red:power`.